changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 15th May 2024
+* Added `SendMarketingPostalMail`, `SendPartnerMarketingEmails` and `SendPartnerMarketingPostalMail` options to [Get all customers](../operations/customers.md#get-all-customers) response.
+* Added `SendMarketingPostalMail`, `SendPartnerMarketingEmails` and `SendPartnerMarketingPostalMail` options to [Update customer](../operations/customers.md#update-customer) resquest.

```

Comment:
```suggestion
* Added `SendMarketingPostalMail`, `SendPartnerMarketingEmails` and `SendPartnerMarketingPostalMail` options to [Update customer](../operations/customers.md#update-customer) request.

```

Reply:
I've make small adjustment, should be fine I hope now 🙏